src/design-system/Loading.tsx

```diff
@@ -1,25 +1,22 @@
-import { css } from "@emotion/core";
+import React from "react";
 import BounceLoader from "react-spinners/BounceLoader";
+import styled from "styled-components";
 
-import { GlobalStyles } from "../styles";
+import Colors from "./Colors";
 
-const override = css`
-  display: block;
-  margin: 0 auto;
-  position: fixed;
-  top: 50%;
-  left: 50%;
-  -ms-transform: translateX(-50%) translateY(-50%);
-  -webkit-transform: translate(-50%, -50%);
-  transform: translate(-50%, -50%);
+const LoadingContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
 `;
 
 const Loading: React.FC = () => {
   return (
-    <>
-      <GlobalStyles />
-      <BounceLoader css={override} size={60} color={"#005450"} />
-    </>
+    <LoadingContainer>
```

Comment:
Another change this PR includes:

Before, all Loading spinners place themselves in middle of the page. This is weird when the middle of the page is already loaded:

![](https://user-images.githubusercontent.com/1570168/78952915-f6ed2180-7a8b-11ea-9887-60067dc3cde9.gif)

Now, the Loading spinner correctly places itself in the middle of the content. So the chart's Loading component is now positioned in the center of the chart!